packages/react-shared-contexts/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "outDir": "lib",
+    "module": "commonjs",
+    "lib": ["es5", "es2015.promise", "dom"],
```

Forgot one thing, we should generally only include promise types if it's used
```suggestion
    "lib": ["es5", "dom"],
```

---

Done!